benefits/settings.py

```diff
@@ -281,7 +288,11 @@ def _filter_empty(ls):
 if len(env_frame_src) > 0:
     CSP_FRAME_SRC = env_frame_src
 
-CSP_IMG_SRC = ["'self'", "data:"]
+CSP_IMG_SRC = [
+    "'self'",
+    "data:",
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c1/Google_%22G%22_logo.svg/1280px-Google_%22G%22_logo.svg.png",
```

Comment:
Just added this here for now, so it'll work out of the box. But we can upload our own Google logo and require it like this: https://megalus.github.io/django-google-sso/customize/#the-button-logo

Reply:
https://github.com/cal-itp/benefits/pull/1855/commits/8df45987b0d3f4e79e1cd182f1250e57b0bf4448